### PR TITLE
Fix broken link to Ingress API

### PR DESCRIPTION
**Pull request by tom1299**
Fix the link to the Ingress API which still had a wrong `apiVersion`

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
/lgtm

I'd prefer a different fix - see inline feedback.

File: content/en/docs/concepts/services-networking/ingress.md [hunk 1]
```diff
@@ -570,6 +570,6 @@ You can expose a Service in multiple ways that don't directly involve the Ingres
 
 ## {{% heading "whatsnext" %}}
 
-* Learn about the [Ingress API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#ingress-v1beta1-networking-k8s-io)
+* Learn about the [Ingress API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#ingress-v1-networking-k8s-io)
```
[the remark this answers is not in the document]
Comment (tom1299) [reply content/en/docs/concepts/services-networking/ingress.md:573]: Absolutely true.

**Comment by sftim** [comment]
/lgtm
/approve

Thanks @tom1299 

Leaving the hold in place, for a different reason: the website is currently frozen, and I'd prefer to unhold this after the rush of post-release changes have gone through.

**Pull request merged**